service/src/main/java/bio/terra/profile/app/configuration/AzureConfiguration.java

```diff
@@ -12,7 +12,7 @@ public record AzureConfiguration(
     String managedAppClientId,
     String managedAppClientSecret,
     String managedAppTenantId,
-    Map<String, AzureApplicationOffer> applicationOffers) {
+    Set<AzureApplicationOffer> applicationOffers) {
```

Comment:
Offers are now exposed as a `Set`, which has the desirable effect of preventing a disconnect between the offer key and the offer name (see change in the test further on in this PR).